### DISCUSSION TITLE
How to show the same module twice with different formats?

### DISCUSSION THREAD

**Discussion by ghost** [Alexays/Waybar#3172] [Q&A] [answered]
Is it possible to have 2 different settings for a module and make it appear twice in the bar?

**Comment by ghost** [comment]
I'm not a programmer but couldn't this be done by using custom module to call another module with specific settings?

**Comment by john-basilio** [comment]
Use custom classes


config.jsonc:
"custom/module-name" {

}

style.css:

"custom/module-name" {

}

**Reply by ghost** [reply]
That doesn't work

**Comment by LukashonakV** [comment, marked as answer]
Hi @not-programmer98, have you seen  [Multiple instances of a module](https://github.com/Alexays/Waybar/wiki/Configuration#multiple-instances-of-a-module) ?

**Reply by ghost** [reply]
Thanks man, That's what I was looking for.
